packages/dialtone-icons/transformSVGtoVue.cjs

```diff
@@ -11,32 +11,38 @@ const _ = require('lodash');
     }
 
     fileNames.forEach(function (fileName) {
-      const svgContent = fs.readFileSync(`./dist/svg/${fileName}`, 'utf8');
-      const template = fs.readFileSync('./src/IconTemplate.vue', 'utf8');
-      const iconName = `dt-icon-${fileName.replace('.svg', '')}`.toLowerCase()
-      .split('-')
-      .map(word => word.charAt(0).toUpperCase() + word.slice(1))
-      .join('');
+      const filePath = `./dist/svg/${fileName}`;
 
-      let result = template.replace('__SVG_CONTENT__', svgContent);
-      result = result.replaceAll('__ICON_NAME__', iconName);
+      if (fs.statSync(filePath).isFile()) { // Check if the item is a file
```

Comment:
`if (fs.statSync(filePath).isFile()) {`

Did this just to by pass my error, I can remove it before merge.